### PR TITLE
Presentation for 2020 ANS Annual meeting

### LINKED ISSUE #24: Add 2020 ANS Annual meeting presentation
This issue can be closed when a first draft of the presentation is added to the repository.

### PR DESCRIPTION
**Pull request by robfairh**
Closes #24. 
This PR contains all the necessary files to build the presentation.

### CODE REVIEW

File: ans-pres/bibliography.bib [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+@misc{us_epa_sources_2020,
+	title = {Sources of {Greenhouse} {Gas} {Emissions}},
+	url = {https://www.epa.gov/ghgemissions/sources-greenhouse-gas-emissions#main-content},
+	language = {en},
+	journal = {Sources of Greenhouse Gas Emissions},
+	author = {US EPA},
+	month = jan,
+	year = {2020}
+}
+
```
Comment (samgdotson) [inline ans-pres/bibliography.bib]:
```suggestion


```
A duplicate, delete this.

**Review by samgdotson — CHANGES_REQUESTED** [review]
Looks good to me, make the one change to the bibliography (it breaks the build) and its good to go

**Comment by robfairh** [comment]
Done

**Comment by samgdotson** [comment]
Perfect, merging

**Pull request merged**